shell/platform/windows/window_proc_delegate_manager_unittests.cc

```diff
@@ -10,11 +10,18 @@ namespace testing {
 
 namespace {
 
+#ifdef _WIN32
+#define FLUTTER_NOINLINE __declspec(noinline)
+#else
+#define FLUTTER_NOINLINE __attribute__((noinline))
+#endif
+
 using TestWindowProcDelegate = std::function<std::optional<
     LRESULT>(HWND hwnd, UINT message, WPARAM wparam, LPARAM lparam)>;
 
 // A FlutterDesktopWindowProcCallback that forwards to a std::function provided
 // as user_data.
+FLUTTER_NOINLINE
```

Comment:
VS 2019 inlines `TestWindowProcCallback` and `TestWindowProcCallback2` together, causing `WindowProcDelegateManagerTest.RegisterMultiple` to fail as the delegates' function pointers were the same:

https://github.com/flutter/engine/blob/8f1c93dec86993043c1f802a9a690b7f8b2d8461/shell/platform/windows/window_proc_delegate_manager_unittests.cc#L95-L119

Reply:
Compilers these days are getting too smart for their own good! 🤣